src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+import java.util.Scanner;
+
+/**
+ * Handles UI related tasks.
+ */
+public class Ui {
+
+    /**
+     * The list of tasks.
+     */
+    private TaskList taskList;
+
+    /**
+     * Creates the Ui object and runs the startup sequence.
+     * It handles all UI related methods
+     */
+    private Ui() {
+        taskList = new TaskList();
+        Storage.read(taskList);
+        greet();
+        listen();
+    }
+
+    /**
+     * Creates a UI object. It is used to start up the chat-bot.
+     */
+    public static void startup() {
+        Ui greg = new Ui();
+    }
+
+    /**
+     * The method used to greet the user upon startup.
+     */
+    public void greet() {
+        fillerLine();
+        System.out.println("    Hello! I am Greg.\n    What can I do for you?");
+        fillerLine();
+    }
+
+    /**
+     * Runs the shutdown sequence.
+     * Saves then says bye.
+     */
+    public void bye() {
```

Comment:
The name of method would be better if it was a verb so maybe use "printBye" or another verb.